### DISCUSSION TITLE
New IP for inverter - where do I change?

### DISCUSSION THREAD

**Discussion by ruteclrp** [wlcrs/huawei_solar#462] [Connecting to the inverter]
My inverter got a new IP. I cannot see which IP it was first set up with in the integration. Any way I can manually update the IP for the integration setup?

**Comment by Roving-Ronin** [comment]
Delete inverter in integration, then re-install.

**Reply by albertsilliker** [reply]
There's any other way? than delete teh integration? My instalation was set with DHCP, and sometimes ip is changed, there's any place to just remove the old one and set the new IP? Thanks

**Comment by Roving-Ronin** [comment]
@ruteclrp Not supported if it stuffs up, at which point you'll need to remove and reinstall the inverter IP via the HA GUI. But, you could try looking in /config/.storage/core.device_registry and /config/.storage/core.config_entries where the IP is stored. Make sure your inverter also has a static IP / DHCP reserved lease in future also.

**Comment by HolgerMiara** [comment]
Isn't it easiest to log-in to your router (which acts as the DHCP server in your home) and to assign "the old" IP address to the dongle again? That will prevent problems in the future as well. Depending on the duration of the lease, it might take some time until the dongle requests a new IP and gets updated (in my router it is 10 days...). You can shorten this period by plugging out your dongle and re-inserting it; the reboot will force it to request a new IP from your router.
